### PR TITLE
some linting 

### PR DESCRIPTION
**Pull request by fperrad**
after merge of https://github.com/libtom/libtommath/pull/123

### CODE REVIEW

File: bn_mp_set_double.c [hunk 1]
```diff
@@ -14,18 +14,18 @@
...
    union {
       double   dbl;
       uint64_t bits;
    } cast;
-   cast.dbl = d;
+   cast.dbl = b;
 
-   exp = (int)(cast.bits >> 52) & 0x7FF;
-   frac = (cast.bits & ((1ULL << 52) - 1)) | (1ULL << 52);
+   exp = (unsigned)(cast.bits >> 52) & 0x7FFU;
```
Comment (sjaeckel) [inline bn_mp_set_double.c]: `exp` should be `unsigned` instead of `int` or not?

Comment (fperrad) [reply bn_mp_set_double.c]: `exp`must stay an `ìnt`

**Comment by sjaeckel** [comment]
thx

File: bn_mp_set_double.c (hunk 1 above)
Comment (fperrad) [reply bn_mp_set_double.c]: àn explicit cast could help to clarify the expression:
```
   exp = (int)((unsigned)(cast.bits >> 52) & 0x7FFU);
```

**Pull request merged**